src/swdmt/redistricting/RedistrictorTest.java

```diff
@@ -226,40 +230,48 @@ public void generateDistrictsSquareContiguityTest() {
         region = new Region();
         districtSet = Redistrictor.generateDistricts(region, 1);
         for (District d : districtSet) {
-            assertTrue("Checking contiguity of district " + d, d.contiguityValid());
+            //assertTrue("Checking contiguity of district " + d, d.contiguityValid());
+            assertTrue(d.contiguityValid());
```

Comment:
The new assertion doesn't provide the additional annotation.  Was that the intent or is this still a "TODO:"?  The original comment doesn't really provide an appropriate message to a tester/maintainer because it doesn't seem phrased as reflecting what was improper about the test failure.

Reply:
When changing 
import static org.junit.Assert.assertThat; to
import static org.junit.jupiter.api.Assertions.assertTrue;

I would get a message that said 
_"Assertions.assertTrue(BooleanSupplier,Supplier<String>) is not applicable (argument mismatch; String cannot be converted to BooleanSupplier)"_
and the tests wouldn't run so I commented out the original tests and replaced them. If you want I can change them to the following which also works.
assertTrue(d.contiguityValid(), "Checking contiguity of district " + d);

Reply:
The syntax of assertions is different in JUnit 4 and JUnit 5.  At the very least, the order of parameters has changed, with the output message parameter moved to be the last parameter.  (This was done to allow lazy evaluation of that parameter.)  You'll need to rewrite assertions to match the JUnit 5 API.